server/reports/commission/commission-report.js

```diff
@@ -1,20 +1,22 @@
-// module "commission-report"
+//module "commission-report"
 import json2csvExports from "json2csv";
+import {printAuthorizationKey, CommissionReport} from "../../commision.js";
```

Comment:
`import CommissionReport from "commission.js";`
Ideally `commission-report.js` should be `commission-handler.js` or `commission-endpoint.js`.